File: 03-frontend-booking-system/booking-system-web/src/app/features/booking/dtos/attendee-booking-response-dto.ts
export class AttendeeBookingResponseDto {
    bookingId!: number;
    attendeeLastName!: string;
    attendeeFirstName!: string;
    isGuest!: boolean;
    bookedStartTime!: string;
    bookedEndTime!: string;
    bookingToken!: string;
    bookedAt!: string;
    bookingStatus!: string;
    eventName!: string;
    eventDescription?: string;
    slotName!: string;
    slotDescription?: string;
    organizerEmail!: string;
    attendeeEmail!: string;
    eventLocationAddress!: string;
    latitude?: number;
    longitude?: number;
}

File: 02-backend-booking-system/src/main/java/com/littlek4za/booking_system/dtos/AttendeeBookingResponseDto.java
package com.littlek4za.booking_system.dtos;

import java.time.Instant;

import com.littlek4za.booking_system.models.BookingStatus;

public record AttendeeBookingResponseDto(
    Long bookingId,
    String attendeeLastName,
    String attendeeFirstName,
    boolean isGuest,
    Instant bookedStartTime,
    Instant bookedEndTime,
    String bookingToken,
    Instant bookedAt,
    BookingStatus bookingStatus,
    String eventName,
    String eventDescription,
    String slotName,
    String slotDescription,
    String organizerEmail,
    String attendeeEmail,
    String eventLocationAddress,
    Double latitude,
    Double longitude) {

}
